secator/celery.py

```
@@ -169,9 +169,10 @@ def run_scan(self, args=[], kwargs={}):
 @app.task(bind=True)
 def run_command(self, results, name, targets, opts={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)